### ISSUE TITLE
AppInsights.TrackEvent() generates name "not_specified"

### ISSUE THREAD

**Issue by michaelsrichter** [IvanJosipovic/BlazorApplicationInsights#12]
When I use the below implementation, App Insights registers the event with name = "not_specified"
```
            await AppInsights.TrackEvent("MyEvent", new Dictionary<string, object>()
            {
                { "customProperty", "customValue" }
            });
```
I believe this is because the javascript needs named parameters, but I'm not sure.

I was able to fix it by defining a class for this:

```
        public class AppInsightsObject
        {
            public string name { get; set; }
            public Dictionary<string, object> properties { get; set; }
        }
```
And then I passed it in like this:
```
            await JSRuntime.InvokeVoidAsync("appInsights.trackEvent", new AppInsightsObject()
            {
                name = "MyEvent",
                properties = new Dictionary<string, object>()
            {
                { "customProperty", "customValue" }
            }
            });
```
This DID work.  I'm not sure I want to submit a full PR, let me know.. 😉

**Comment by IvanJosipovic** [comment]
Were you using version 1.13.1 in your tests? I fixed some JS Interop logic in that version.

**Comment by IvanJosipovic** [comment]
This issue is fixed in release [0.5.0](https://github.com/IvanJosipovic/BlazorApplicationInsights/releases/tag/v0.5.0)

**Comment by michaelsrichter** [comment]
@IvanJosipovic thanks, I confirmed it's working now.
